python-frontend/src/main/java/org/sonar/python/semantic/v2/FunctionTypeBuilder.java

```diff
@@ -148,8 +148,16 @@ private void addParameter(org.sonar.plugins.python.api.tree.Parameter parameter,
     Token starToken = parameter.starToken();
     if (parameterName != null) {
       ParameterType parameterType = getParameterType(parameter);
-      this.parameters.add(new ParameterV2(parameterName.name(), parameterType.pythonType(), parameter.defaultValue() != null,
-        parameterState.keywordOnly, parameterState.positionalOnly, parameterType.isKeywordVariadic(), parameterType.isPositionalVariadic(), locationInFile(parameter, fileId)));
+      var parameterV2 = new ParameterV2(parameterName.name(),
```

Comment:
Do we need the changes regarding `FunctionType` building for this PR?
I feel we can focus on the builtins fallback which would then be clearly separated from this.

Reply:
excluded